editor/renames_map_3_to_4.cpp

```diff
@@ -574,22 +574,23 @@ const char *RenamesMap3To4::gdscript_function_renames[][2] = {
 	{ "get_uniform_name", "get_parameter_name" }, // ParameterRef
 
 	// Builtin types
-	// Remember to add them to builtin_types_excluded_functions variable, because for now this functions cannot be listed
-	//	{ "empty", "is_empty" }, // Array - Used as custom rule  // Be careful, this will be used everywhere
-	{ "clamped", "clamp" }, // Vector2  // Be careful, this will be used everywhere
+	// Remember to add them to the builtin_types_excluded_functions variable, because for now these functions cannot be listed.
+	//	{ "empty", "is_empty" }, // Array -- Used as custom rule. Be careful, this will be used everywhere.
+	{ "clamped", "clamp" }, // Vector2 -- Be careful, this will be used everywhere.
 	{ "get_rotation_quat", "get_rotation_quaternion" }, // Basis
 	{ "grow_margin", "grow_side" }, // Rect2
-	{ "invert", "reverse" }, // Array - TODO check  // Be careful, this will be used everywhere
+	{ "invert", "reverse" }, // Array -- Give it a check. Be careful, this will be used everywhere.
 	{ "is_abs_path", "is_absolute_path" }, // String
 	{ "is_valid_integer", "is_valid_int" }, // String
 	{ "linear_interpolate", "lerp" }, // Color
 	{ "find_last", "rfind" }, // Array, String
+	{ "remove", "remove_at" }, // Array -- Breaks Directory and several more.
 	{ "to_ascii", "to_ascii_buffer" }, // String
 	{ "to_utf8", "to_utf8_buffer" }, // String
-	{ "to_wchar", "to_utf32_buffer" }, // String // TODO - utf32 or utf16?
+	{ "to_wchar", "to_utf32_buffer" }, // String -- utf32 or utf16?
```

Comment:
That TODO should be resolved probably. @bruvzg

Reply:
It depends on OS, `to_utf16_buffer` if it's Windows, `to_utf32_buffer` if not. Maybe we should expose `to_wchar_buffer` as well?

Reply:
https://github.com/godotengine/godot/pull/73225